missions/ME_Maya_Missions.txt

```diff
@@ -2595,10 +2607,24 @@ maya_missions_5 = {
 		icon = me_mission_mayas_temple
 		required_missions = { maya_ahkinob }
 		position = 4
-		trigger = {
-			OR = {
-				cathedral = 10
-				temple = 30
+			trigger = {
+			calc_true_if = {
+				all_owned_province = {
+					custom_trigger_tooltip = {
+						tooltip = PROVINCE_HAS_BUILDING_OF_GROUP_TAXATION_ALL
+						province_has_building_of_group = { group = taxation all = yes }
+					}
+				}
+				amount = 30
+			}
+			calc_true_if = {
+				all_owned_province = {
+					custom_trigger_tooltip = {
+						tooltip = PROVINCE_HAS_BUILDING_OF_GROUP_TAXATION_EOMAT_cathedral
+						province_has_building_of_group = { group = taxation equal_or_more_advanced_than = cathedral }
+					}
+				}
+				amount = 10
```

Comment:
These need to be in an OR?

Reply:
Oh! I misinterpreted the original. Thought it required 30 temples and 10 cathedrals.